shared/candidate-helpers.js

```diff
@@ -1,5 +1,28 @@
+var _ = require('underscore');
 var committeeHelpers = require('./committee-helpers.js');
```

Comment:
I don't think we need this line. I think I originally put it in there because I was using the committeeHelpers.buildCommitteeContext() function but no more.